ob1k-cache/src/main/java/com/outbrain/ob1k/cache/LocalAsyncCache.java

```diff
@@ -114,7 +118,8 @@ public LocalAsyncCache() {
       return fromValue(res);
     } else {
       if (failOnMissingEntries) {
-        return fromError(new RuntimeException("result is missing from loader response."));
+        final String prefix = cacheName == null ? "" : cacheName + ": ";
+        return fromError(new RuntimeException(prefix + "result for " + key + " is missing from loader response."));
```

Comment:
I guess my main point you can't assume such, this also can lead to really huge exceptions if the key is complex type, which toString of that would produce big multi-line block.

this may be moved to a log message, but not as part of exception message